opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -261,13 +238,27 @@ namespace Opm
         // untill the number of closed completions do not increase anymore.
         while (testWell) {
             const size_t original_number_closed_completions = welltest_state_temp.sizeCompletions();
-            solveWellForTesting(simulator, well_state_copy, group_state, deferred_logger);
+            bool converged = solveWellForTesting(simulator, well_state_copy, group_state, deferred_logger);
+            if (!converged) {
+                const std::string msg = " well " + this->name() + " did not get converged during well testing";
+                deferred_logger.debug(msg);
+                return;
+            }
+
+            updateWellOperability(simulator, well_state_copy, deferred_logger);
+            if ( !this->isOperable() ) {
+                const std::string msg = " well " + this->name() + " is not operable during well testing for physical reason";
```

Comment:
```C++
auto msg = fmt::format("Well {} is not operable during well testing for physical reason", this->name());
```

Reply:
If we do update this, can we rephrase the message to be less ambiguous too?  E.g, something along the lines of

> WTEST: Well &lt;name&gt; is not operable (physical reasons)